live-examples/html-examples/css/datalist.css

```diff
@@ -0,0 +1,3 @@
+label, input {
+  font-size: 1.2em;
```

Comment:
Nit, code style: indent with 4 spaces. I believe we should stick to `rem` units for font sizes?. This would be the recommendation in the code style guide I am working on.